_posts/2020-07-22-#15644.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: pr
+date: 2020-07-22
+title: "Make orphan processing interruptible"
+pr: 15644
+authors: [sipa]
+components: ["p2p"]
+host: narula
+status: upcoming
+commit:
+---
+
+
+## Notes
+
+* [PR 15644](https://github.com/bitcoin/bitcoin/pull/15644) was merged
+  over a year ago. It changes the way a node looks through its orphan
+  set to figure out if orphans can now be processed and added to the
+  mempool.
+
+* A transaction is an _orphan_ if we have not yet seen any of its
+  inputs. This might be a valid transaction, but we don't know yet. We
+  have to be careful with orphan management because orphans cannot be
+  validated.
```

Comment:
> Perhaps mention that handling orphan transactions can be fertile ground for memory/CPU exhaustion attacks.

I don't think that needs explicitly spelling out here, since we'll probably cover this in depth in the meeting.

> A couple of resources we could link to in the notes...

Great. I've added these.